firrtl/src/main/scala/firrtl/ir/IR.scala

```diff
@@ -372,28 +371,32 @@ case class PropAssign(info: Info, loc: Expression, expr: Expression) extends Sta
 case class IsInvalid(info: Info, expr: Expression) extends Statement with HasInfo with UseSerializer
 case class Attach(info: Info, exprs: Seq[Expression]) extends Statement with HasInfo with UseSerializer
 
-@data class Stop(info: Info, ret: Int, clk: Expression, en: Expression, @since("FIRRTL 1.5") name: String = "")
+case class Stop(
+  val info: Info,
+  val ret:  Int,
+  val clk:  Expression,
+  val en:   Expression,
+  val name: String = "")
     extends Statement
     with HasInfo
     with IsDeclaration
     with UseSerializer {
   def copy(info: Info = info, ret: Int = ret, clk: Expression = clk, en: Expression = en): Stop = {
-    Stop(info, ret, clk, en, name)
+    new Stop(info, ret, clk, en, name)
   }
 }
 object Stop {
   def unapply(s: Stop): Some[(Info, Int, Expression, Expression)] = {
     Some((s.info, s.ret, s.clk, s.en))
   }
 }
```

Comment:
The custom `copy` and `unapply` are there to preserve binary compatibility (which is why we used `@data` in the first place. We should just break binary compatibility and go ahead and delete the manual copy and unapply as well. Same for `Print` below.